apps/src/applab/applab.js

```diff
@@ -6,7 +6,7 @@
  */
 
 'use strict';
-
+require('./mode-javascript_codeorg');
```

Comment:
require this file so that browserify knows about it and bundles it with applab.js